install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+function install {
+  printf "\e[92m[~] Name your project: \e[0m\n"
+
+  read name
+
+  printf "\e[33m[~] Downloading awesomeness...\e[0m\n"
+
+  git clone https://github.com/mostjs/package-starter $name \
```

Comment:
Do we need parens or quotes around `$name`, e.g. `$(name)` or `"$name"` in case the user types a name with a space in it?  Or maybe we should disallow names with spaces?